model/auth.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+const (
+	// IstioAuthCACertificate defines the name for CA root certificate file.
+	IstioAuthCACertificate = "root-cert.pem"
```

Comment:
quick comment: can we use a more descriptive name for the in-cluster mutual TLS rather than "Istio Auth". Let's drop Istio to reduce stutter (all code has istio.io in the name already), and clarify which "auth" this refers to.